### ISSUE TITLE
Buck invocation on java project is failing with 'cxx linker type detection' error

### ISSUE THREAD

**Issue by davido** [facebook/buck#243]
On most recent mater (0784a631c7d0742dd74e30cd5513ea37b992d7d4), on other (non `Linux`) platform, trying to use `buck` for a `java` project (https://github.com/davido/buck_test) is failing with:

```
BUILD FAILED: cannot detect linker type, try explicitly setting it in .buckconfig's [cxx] ld_type section
```
This diff fixed this:
```
diff --git a/src/com/facebook/buck/cxx/DefaultCxxPlatforms.java b/src/com/facebook/buck/cxx/DefaultCxxPlatforms.java
old mode 100644
new mode 100755
index 32bfaa2..7061153
--- a/src/com/facebook/buck/cxx/DefaultCxxPlatforms.java
+++ b/src/com/facebook/buck/cxx/DefaultCxxPlatforms.java
@@ -160,6 +160,7 @@ public class DefaultCxxPlatforms {
   private static LinkerType getLinkerTypeForPlatform(Platform platform) {
     switch (platform) {
       case LINUX:
+      case WINDOWS:
         return LinkerType.GNU;
       case MACOS:
         return LinkerType.DARWIN;

**Comment by sdwilsh** [comment]
I fixed this; I just forgot to do a push this week.  I'll start one now.

**Comment by sdwilsh** [comment]
I think this was fixed with 6037895d56a5beca65faefadf0165d7c5cda97a6

**Comment by davido** [comment]
Yes it did. Thanks.
